aspnetcore/mvc/controllers/areas/60samples/RPareas/Areas/Products/Pages/About.cshtml.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Docs.Samples;
+
+namespace RPareas.Areas.Products.Pages;
+
+public class AboutModel : PageModel
+{
+    public ContentResult OnGet() =>
+     Content(PageContext.MyDisplayRouteInfoRP().ToString()!);
```

Comment:
@serpent5 I get this from my NuGet package https://github.com/Rick-Anderson/RouteInfo/blob/master/Microsoft.Docs.Samples.RouteInfo/PageContextExtensions.cs

How can I get rid of `ToString()`